object/address/address.go

```diff
@@ -10,6 +10,14 @@ import (
 )
 
 // Address represents v2-compatible object address.
```

Comment:
```suggestion
// Address defines the address space for NeoFS objects.
```

Reply:
why? thought all public types should have such info